bots/discord/sinusbot/README.md

```diff
@@ -2,7 +2,7 @@
 ### Their [Site](https://www.sinusbot.com/)
 Listen to your favorite music together with all of your friends  
 
-Welcome the Simple, Elegant & great sounding TS3- and Discord-Bot!  
+Welcome the Simple, Elegant & great sounding TS3 and Discord-Bot!  
```

Comment:
This was copied directly from their website and I would prefer we leave it that way.